test/test/cpu/CpuTests.java

```diff
@@ -23,7 +24,9 @@ private static void assertCloseTo(long value, long target, String message) {
 
     private static void pinCpu(TestProcess p, int cpu) throws Exception {
         String[] tasksetCmd = {"taskset", "-acp", String.valueOf(cpu), String.valueOf(p.pid())};
-        ProcessBuilder cpuPinPb = new ProcessBuilder(tasksetCmd).inheritIO();
+        ProcessBuilder cpuPinPb = new ProcessBuilder(tasksetCmd)
+            .redirectError(Redirect.INHERIT)
+            .redirectOutput(Redirect.DISCARD);
```

Comment:
`Redirect.DISCARD` is JDK 9+ only.
Maybe `redirectOutput(new File("/dev/null"))` ?

Reply:
Ops, f0b4f25cd1dff941f6156bf8244141f3f39f781e